trunk-recorder/systems/p25_parser.cc

```diff
@@ -421,6 +453,9 @@ std::vector<TrunkMessage> P25Parser::decode_tsbk(boost::dynamic_bitset<> &tsbk,
     message.source = sa;
     message.emergency = emergency;
     message.encrypted = encrypted;
+    message.duplex = duplex;
+    message.mode = mode;
+    message.priority = priority;
```

Comment:
From the [buildx @ 86689ce](https://github.com/robotastic/trunk-recorder/runs/2624228538#step:7:3420):
```
#13 300.5 /src/trunk-recorder/systems/p25_parser.cc:456:22: error: 'duplex' was not declared in this scope
#13 300.5   456 |     message.duplex = duplex;
#13 300.5       |                      ^~~~~~
#13 300.5 /src/trunk-recorder/systems/p25_parser.cc:457:20: error: 'mode' was not declared in this scope; did you mean 'modf'?
#13 300.5   457 |     message.mode = mode;
#13 300.5       |                    ^~~~
#13 300.5       |                    modf
#13 300.5 /src/trunk-recorder/systems/p25_parser.cc:458:24: error: 'priority' was not declared in this scope
#13 300.5   458 |     message.priority = priority;
#13 300.5       |                        ^~~~~~~~
```

@rosecitytransit do [these lines](https://github.com/robotastic/trunk-recorder/pull/470/files#diff-f75ab21e8810348f73654b768283e40a0d9948d9a1d89c12cd73fd55c5b6cba4R442-R444) need to be uncommented out?

Reply:
Yep, they did. I was thinking it should have been done in the patch I applied in rebuilding to create this PR, but it looks like I may have never have changed those lines.